src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -130,25 +167,31 @@ public ICustomToolTip GetToolTip(string name, string group = "general")
     /// <summary>
     ///   Creates a new group node to contain tooltips
     /// </summary>
-    public void AddGroup(string name)
+    public Control AddGroup(string name)
     {
+        GD.Print("Creating new tooltip group: '" + name + "'");
```

Comment:
When does this trigger? I didn't see this happen in my tests.

Reply:
`AddGroup` is called here inside `AddToolTip():` https://github.com/Revolutionary-Games/Thrive/blob/8dc4518f7e3b64664f64260271d4b38acbc8ed76/src/gui_common/tooltip/ToolTipManager.cs#L104
And that will be called when creating tooltips for the data points and legend.

Reply:
After reading all of the PR it now seems that only the graph control registers new tooltips.